src/development/platform-integration/ios/apple-frameworks.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Leveraging Apple's System Libraries
+description: Learn how to apply knowledge of Apple's APIs and frameworks when building Flutter apps.
+---
+
+This document is for iOS developers looking to apply
+their existing knowledge of Apple's APIs within Flutter.
+This includes accessing the device hardware and interacting 
+with specific frameworks like `HealthKit` or `MapKit`.
+
+<!-- Add once SwiftUI PR is merged -->
+<!-- For an overview of how the SwiftUI framework compares to Flutter, 
+see [Learning Flutter as a SwiftUI developer][].  -->
+
+### Flutter plugin overview
+When developing an app with Flutter, you can interact 
+with system libraries using plugins. 
+Plugins are Dart libraries that contain platform-specific code. 
+In the case of an iOS app, 
+the platform code can be written in Objective-C or Swift. 
+
+In your Dart code, you use the plugin's Dart API and 
+behind the scenes the native plugin code will directly call 
+the system library being used. This means that you can write 
+the code to call the Dart API once, 
+and it will work for all the platforms 
+that the plugin has been written for. 
+
+To learn more about plugins see [Using packages][]. 
+This document links out to some commonly used plugins, 
+and their examples, but there are thousands more available on [pub.dev][]. 
+And, if you can't find what you're looking for, you can create your own. 
```

Comment:
**suggestion:** Tighten up this language.
```suggestion
Though this page links to some popular plugins,
you can find thousands more on [pub.dev][]. 
If you can't find a package that meets your need, 
you can create your own. 
```

Reply:
I want to make it clear that there are examples on pub.dev as well